pkg/transaction-builder/models.go

```diff
@@ -0,0 +1,124 @@
+package transactionbuilder
+
+import (
+	"encoding/hex"
+
+	"github.com/pokt-network/pocket-core/crypto"
+	coreTypes "github.com/pokt-network/pocket-core/types"
+	appsType "github.com/pokt-network/pocket-core/x/apps/types"
+	nodesTypes "github.com/pokt-network/pocket-core/x/nodes/types"
+)
+
+// NewMsgSend returns message for send transaction
+func NewMsgSend(fromAddress, toAddress string, amount int64) (TxMsg, error) {
```

Comment:
A bit of a bikeshed here, but if we're not including `Msg` in all of the other methods, we shouldn't include it here either. 
```suggestion
func NewSend(fromAddress, toAddress string, amount int64) (TxMsg, error) {
```